strawberry/static/graphiql.html

```diff
@@ -32,7 +32,7 @@
     <script src="https://cdn.jsdelivr.net/npm/graphiql-with-extensions@0.14.3/graphiqlWithExtensions.min.js"
         integrity="sha384-TqI6gT2PjmSrnEOTvGHLad1U4Vm5VoyzMmcKK0C/PLCWTnwPyXhCJY6NYhC/tp19"
         crossorigin="anonymous"></script>
-    <script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.min.js"></script>
+    <script src="https://cdn.jsdelivr.net/npm/js-cookie@rc/dist/js.cookie.umd.min.js"></script>
```

Comment:
I think it would be better to pin the version to a specific rc version like the suggestion here: https://github.com/js-cookie/js-cookie/issues/698#issuecomment-880242681